### ISSUE TITLE
Val accuracy drop with -trainBatch increasing

### ISSUE THREAD

**Issue by weiguochow** [xingyizhou/pytorch-pose-hg-3d#15]
Hi, Xingyi
I enlarged the training batch size in order to accelerate the training process. On the contrary, the validation accuracy was dropping.  Thank you!!!
